keyboards/thevankeyboards/minivan/keymaps/josjoha/unicode_macros.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * License (GPL):
+  
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ * Authors: This QMK keymap file is a combination of the default
+ * keymap, led code original copied/edited from ../jetpacktuxedo, some
+ * copy/paste from QMK documentation code examples (etc).
+ * Remainder: (C) 2019 by J.B. <joshb@xs4all.nl>
+ *
+ */
+
```

Comment:
```suggestion
#pragma once

```

Reply:
Thanks for reviewing, I'll incorporate it once I can. Still doing some minor things, then I'll try an upload.
I wouldn't recommend digging through the keymap at this time, because so much has changed.